src/OptionsStruct.jl

```diff
@@ -59,7 +125,7 @@ struct Options{A,B,dA,dB,C<:Union{SupervisedLoss,Function},D}
     annealing::Bool
     batching::Bool
     batchSize::Int
-    mutationWeights::Array{Float64,1}
+    mutation_weights::MutationWeights
```

Comment:
Good point! I’ll leave the old name for now then. Hopefully I’ll do a big refactoring soon and change the name cases

Reply:
Okay I decided to instead just refactor everything now to be in snake case. I set it up so that all the old parameters still work (and produce a `depwarn`), so it should be fine.